SVGEngine.mm

```diff
@@ -125,7 +133,9 @@ @interface SVGAttributeSet () {
                 pushGroup(readAttributes());
             else if(type == XML_READER_TYPE_END_ELEMENT)
                 popGroup();
-        }
+        } else if(type == XML_READER_TYPE_ELEMENT)
```

Comment:
Please use curlies `{}`

Reply:
Additionally this check should just be `if (type == XML_READER_TYPE_ELEMENT && !xmlTextReaderIsEmptyElement(_xmlReader))` skipping the next if statement

Reply:
Since this is changing to be a single-line branch I presume that you don't want curlies around it, in order to match the rest of the code here?